src/App.jsx

```diff
@@ -1,40 +1,33 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
 
+import { Tabs } from './components/Tabs/Tabs';
```

Comment:
```suggestion
import { Tabs } from './components/Tabs';
```
Bring back `index.js` before doing this.

Reply:
@desevoker comment makes sense